internal/monitors/conviva/configsetter.go

```diff
@@ -0,0 +1,153 @@
+package conviva
+
+import (
+	"context"
+	"strconv"
+	"strings"
+)
+
+func setConfigFields(m *Monitor, conf *Config)  {
+	conf.metriclensMetricNames = map[string][]string{
+		"quality_metriclens": {
+			"total_attempts",
```

Comment:
Some of these sound like counters, not gauges.  Why are they all treated as gauges?

Reply:
You would think that but looking at the total_attempts plot in pulse (conviva's web portal), total_attempts is not monotonically increasing!